File: modules/jpa/testdata/inspection/constructorExpression/src/Owner.java
import jakarta.persistence.*;

@Entity
public class Owner {
    public Owner() {}
    public Owner(String name) {
        this.name = name;
    }
    
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    
    @Column
    private String name;
    
    public Long getId() {
        return id;
    }
    
    public void setId(Long id) {
        this.id = id;
    }
    
    public String getName() {
        return name;
    }
    
    public void setName(String name) {
        this.name = name;
    }
}

File: modules/jpa/testdata/kotlin/inspection/constructorExpression/src/Owner.kt
import jakarta.persistence.*

@Entity
class Owner {
    constructor()
    constructor(name: String?) {
        this.name = name
    }

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    var id: Long? = null

    @Column
    var name: String? = null
}
